### PR TITLE
add definion for projectsconfig

### PR DESCRIPTION
**Pull request by zhouying7780**

### CODE REVIEW

**Comment by zhouying7780** [comment]
@kasturinarra Please help review , thanks

File: features/step_definitions/crd.rb [hunk 1]
```diff
@@ -9,3 +9,23 @@
     raise "#{crd} not supported by this step"
   end
 end
+
+Given /^the #{QUOTED} #{QUOTED} CR is restored after scenario$/ do |name,crd|
```
Comment (kasturinarra) [inline features/step_definitions/crd.rb:13]: space between name,crd ?

**Comment by kasturinarra** [comment]
@zhouying7780  looks good to me, just a minor comment, can you please help take a look , thanks !!

File: features/step_definitions/crd.rb (hunk 1 above)
Comment (zhouying7780) [reply features/step_definitions/crd.rb:13]: @xingxingxia  CR or CRD which one is better ?

Comment (xingxingxia) [reply features/step_definitions/crd.rb:13]: CR. Mentioned in https://github.com/openshift/cucushift/pull/7479#pullrequestreview-378194597 .

**Comment by zhouying7780** [comment]
@liangxia Could you help review and merge , thanks .

**Comment by zhouying7780** [comment]
@liangxia I've resolved all the comments , please help review , thanks .

**Comment by zhouying7780** [comment]
@kasturinarra Please help review , thanks

**Comment by kasturinarra** [comment]
@liangxia could you please help review and merge ? Thanks !!

**Pull request merged**